src/query/execute_schema.c

```diff
@@ -6952,6 +6971,44 @@ do_promote_partition_by_name (const char *class_name, const char *part_num, char
   return NO_ERROR;
 }
 
+/*
+ * has_notnull_unique_constraints() - Check if attribute is NOT NULL and UNIQUE
+ *    return: true if the attribute has both NOT NULL and UNIQUE constraints,
+ *            false otherwise
+ *    smattr(in): attribute to check
+ *
+ * Note:
+ *    The decision is based on both smattr->flags (for NOT NULL)
+ *    and smattr->constraints (for UNIQUE).
+ *    - If the attribute has both NOT NULL and UNIQUE constraints, return true
+ *      so that the UNIQUE flags are preserved.
+ *    - Otherwise, return false and the UNIQUE-related flags will be reset.
+ */
+static bool
+has_notnull_unique_constraints (const SM_ATTRIBUTE * smattr)
+{
+  bool has_not_null = false;
+  bool has_unique = false;
+  SM_CONSTRAINT *c;
+
+  has_not_null = (smattr->flags & SM_ATTFLAG_NON_NULL) != 0;
+  if (!has_not_null)
+    {
+      return FALSE;
```

Comment:
NIT: bool 변수값 스타일 일관성
```suggestion
      return false;
```

Reply:
@hornetmj 반영했습니다.